src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -435,5 +522,90 @@ public async Task SslStream_ClientCertificate_SendsChain()
             // Accept any certificate.
             return true;
         }
+
+        private class RandomIOStream : Stream
```

Comment:
The name "RandomIOStream" isn't very descriptive; it sounds like we are generating random data, which is not at all what this is doing.

Can we call it something more descriptive? Like maybe "RandomReadWriteSizeStream"?

Reply:
And this seems like it could be generally useful, should we split it into a separate file, maybe put in Common, etc?

Reply:
Also same comment re naming for the test name itself -- "SslStream_RandomWrites_OK"